web/client-api/src/main/java/io/deephaven/web/client/api/CoreClient.java

```diff
@@ -134,13 +144,8 @@ public Promise<Void> onConnected(@JsOptional Double timeoutInMillis) {
         return ideConnection.onConnected();
     }
 
-    public Promise<String[][]> getServerConfigValues() {
-        return getConfigs(
-                c -> ideConnection.connection.get().configServiceClient().getConfigurationConstants(
-                        new ConfigurationConstantsRequest(),
-                        ideConnection.connection.get().metadata(),
-                        c::apply),
-                ConfigurationConstantsResponse::getConfigValuesMap);
+    public Promise<JsServerConfigValues> getServerConfigValues() {
```

Comment:
We can't just change the API, as that will break existing consumers. Of course, doing this change would now match how it is on Enterprise...